simplq/src/store/asyncActions/deleteQueue.js

```diff
@@ -11,11 +14,20 @@ const typePrefix = 'deleteQueue/action';
  */
 const useDeleteQueue = () => {
   const makeAuthedRequest = useMakeAuthedRequest();
+  const dispatch = useDispatch();
+  const history = useHistory();
 
   const deleteQueue = createAsyncThunk(typePrefix, async (arg) => {
     const { queueId } = arg;
     const authedRequest = makeAuthedRequest(RequestFactory.deleteQueue(queueId));
     const response = await authedRequest;
+    if (arg.goHome) {
+      history.push('/');
+    }
+    if (arg.popUp) {
+      dispatch(setInfoPopupMessage('Successfully deleted queue'));
+    }
```

Comment:
Message handling can certainly be done like this. The advantage is that it looks synchronous and it is obvious what happens and when.

It can also be done in `appSlice`, since `InfoPopup` will fire on every local state change. The advantage is that we can react on `pending`, `fulfilled` and `rejected` events if/when needed.